ionoscloud/resource_server_cube.go

```diff
@@ -352,6 +360,15 @@ func resourceCubeServer() *schema.Resource {
 					},
 				},
 			},
+			//Keep track of the inline volume of the Cube server in case the boot volume is changed.
+			"inline_volume_ids": {
```

Comment:
Can you test an upgrade pre to  after inline_volume_ids? I think we need to add  something similar to resoruce_server
	// takes care of an upgrade from a version that does not have inline_volume_ids(pre 6.4.0)
	// to one that has it(>6.4.0)
	if _, ok := d.GetOk("inline_volume_ids"); !ok {
		if bootVolumeItf, ok := d.GetOk("boot_volume"); ok {
			bootVolume := bootVolumeItf.(string)
			var inlineVolumeIds []string
			inlineVolumeIds = append(inlineVolumeIds, bootVolume)
			if err := d.Set("inline_volume_ids", inlineVolumeIds); err != nil {
				return utils.GenerateSetError("server", "inline_volume_ids", err)
			}
		}
	}